### DISCUSSION TITLE
CompilationError in task app:inferDebugAndroidTestKeepRulesForKeeper

### DISCUSSION THREAD

**Discussion by JacobKant** [slackhq/keeper#102] [Q&A]
I can't figure out what the problem is and how to fix it (

version 0.11.2
i use databinding, hilt, dagger 
the error appears about the classes that are generated, it happens with dagger or databinding

:app:inferDebugAndroidTestKeepRulesForKeeper
Exception in thread "main" com.android.tools.r8.errors.CompilationError: Class content provided for type descriptor \com\myapp\databinding\LiLogEmptyBindingImpl ac
tually defines class com.myapp.databinding.LiLogEmptyBindingImpl

**Comment by ZacSweers** [comment]
This is quite vague and not enough information to help you. Please make a minimally reproducible sample project.
